recipes/bgfx/config.yml

```diff
@@ -1,3 +1,5 @@
 versions:
+  "1.127.8771":
+    folder: consolidated
   "cci.20230216":
     folder: all
```

Comment:
I think it might be better to put the new recipe in all since it will be all moving forward, and to move the old recipe to another folder name like `cci.2023`

Reply:
We can move forward with this, I also think it's cleaner

Reply:
By "this" we mean moving the new stuff to "all"?

Reply:
Yes :) and the old recipe to a folder with a clarifying name, I'll let you come up with it

Reply:
Circling back to this, is there any actual reason to keep the cci version around? I think it would be ok to just keep the released versions from now on